votes/0639-propose-a-release-note-guide.md

```diff
@@ -0,0 +1,76 @@
+| copr: cast invalid utf8 string to real bug fix               | ❌                 | ❌                                 | ❌                    | Before v4.0.16, when TiDB converts an illegal UTF-8 string to a Real type, an error is reported directly. Starting from v4.0.16, TiDB processes the conversion according to the legal UTF-8 prefix in the string |
+| sink: fix kafka max message size inaccurate issue            | ❌                 | ❌                                 | ❌                    | Change the default value of Kafka Sink max-message-bytes to 1 MB to prevent TiCDC from sending too large messages to Kafka clusters |
+| cdc/sink: adjust kafka initialization logic                  | ❌                 | ❌                                 | ❌                    | Change the default value of Kafka Sink partition-num to 3 so that TiCDC distributes messages across Kafka partitions more evenly |
+| cmd: hide --sort-dir in changefeed command. (deprecated warning exists) | ✅                 | ❌                                 | ❌                    | Deprecate `--sort-dir` in the `cdc cli changefeed` command. Instead, users can set `--sort-dir` in the `cdc server` command. |
+
+## Bug fix
+
+A bug fix note means that your PR fixes an existing bug or issue. This type of notes start with "Fix" followed by "the issue/bug".
+
+Write your note clearly and adequately so that your target readers can get the main point of your bug fix. The bug or issue must be directly perceivable to the users, and you can refer to the associated GitHub issues.
+
+**Examples:**
+
+| **Not recommended**                                          | **Clear in type** | **Adequate and clear in meaning** | **User perspective** | Recommended                                                  |
+| ------------------------------------------------------------ | ----------------- | --------------------------------- | -------------------- | ------------------------------------------------------------ |
+| lock_resolver: avoid pessimistic transactions using resolveLocksForWrite | ❌                 | ✅                                 | ❌                    | Fix the issue that committing pessimistic transactions might cause write conflict |
+| retry when meeting stablish conn fails                       | ❌                 | ❌                                 | ❌                    | Fix the issue of unexpected results when TiFlash fails to establish MPP connections |
+| Fix the issue that greatest(datetime) union null returns empty string | ✅                 | ❌                                 | ✅                    | Fix the issue that the query result might be wrong when NULL is in the UNION subquery |
+| copr: make CM Sketch built with the same encoding as what TiDB assumes | ❌                 | ❌                                 | ❌                    | Fix the wrong statistics caused by the format inconsistency of CM Sketch between TiKV and TiDB |
```

Comment:
From the user perspective we should describe when this problem will happen. It is what user really cares about. The CM Sketch is not a concept exposed to user so that it should not be mentioned.

```suggestion
| copr: make CM Sketch built with the same encoding as what TiDB assumes | ❌                 | ❌                                 | ❌                    | Fix wrong statistics when `tidb_analyze_version` is set to `1` |
```

Reply:
I see that all bug fix notes are actually prefixed with "Fix the issue that ...". Is it possible to be simplified? For example, how about "Fix the analyzed statistics might be wrong when `tidb_analyze_version` is set to `1`"?

Reply:
I don't think "Fix the analyzed statistics might be" is grammatically correct. 

"Fix the issue" can have some variations, such as "Fix the issue of potential wrong analyzed statistics when ...".